packages/storybook/src/stories/ino-switch/ino-switch.stories.ts

```diff
@@ -27,9 +28,16 @@ export default {
       return story();
     },
   ],
-} as Meta;
+  args: {
+    checked: false,
```

Comment:
Think it would be nicer if the default state is switched on
```suggestion
    checked: true,
```